source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,233 @@ Hardware Checklist
    :local:
    :depth: 2
 
-Stub - detail Hardware Checklist Items
+Use the following checklist when planning the hardware configuration for a production, distributed MinIO deployment.
+
+Considerations
+--------------
+
+When selecting hardware for your MinIO implementation, take into account the following factors:
+
+- Expected amount of data in tebibytes to store at launch
+- Expected growth in size of data for at least the next two years
+- Number of objects by average object size
+- Average retention time of data in years
+- Number of sites to be deployed
+
+Hardware Requirements and Recommendations
+-----------------------------------------
+
+.. note:: 
+
+   See our `Reference Hardware <https://min.io/product/reference-hardware?ref-docs>`__ page for a curated selection of servers and storage components from our hardware partners.
+
+   MinIO does not provide hosted services or hardware sales.
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - | Sufficient CPU cores to acheive performance goals for hashing (for example, for healing) and encryption
+       | MinIO recommends Dual Intel® Xeon® Scalable Gold CPUs (minimum 8 cores per socket) or any CPU with AVX512 instructions
```

Comment:
AFAIK avx512 is a nice to have, but since a) we don't use much CPU regardless and b) the performance gains for avx512 vs avx2 are not enough to make things be mandatory to have avx512 in play. I sort of waffle on this since, no, it is not needed strictly, but on the other side, if you have an Intel processor that doesn't have 512, it might also indicate the age of the hardware could have other issues (AMD is a different story since they do not have avx512 or didn't introduce it until recently). So all that a very long winded way to ask if mandating 512 causes more confusion than it solves?

Reply:
We're not mandating. We're recommending.
> **MinIO recommends** .... or any CPU with AVX512 instructions.

Do you think that is sufficient, @eco-minio? Or do we just need to remove that.

Reply:
I just want to mitigate the conversation of "but, I only have avx2, am I doomed?" but probably for the amount of times we have to clarify it might not be worth the effort to get verbiage around. I would stick with what you have for now and we can amend in the future if it ever was needed

Reply:
Maybe if we trim the H1 to just "Hardware Recommendations", since we do have "Requirements and Recommendations"

That said, we're allowed to be prescriptive. We could try to address this in the lede

```
Hardware Requirements
--------------------------------

The following table provides a minimum hardware specification for production MinIO deployments. 
While MinIO can run on commodity or "budget" hardware, we strongly recommend using this table as guidance for best results in production environments.
```